gbpservice/nfp/service_vendor_agents/vyos/vyos_session/utils.py

```diff
@@ -68,6 +69,13 @@ def init_logger(logger):
     # add the handlers to the logger
     logger.addHandler(file_handler)
 
+    formatter = logging.Formatter('vyos %(name)s %(funcName)s() %(levelname)s '
+                                  '%(message)s')
+    sys_handler = handlers.SysLogHandler(address=('localhost', 514))
+    sys_handler.setFormatter(formatter)
+    sys_handler.setLevel(logging.DEBUG)
+    logger.addHandler(sys_handler)
```

Comment:
2 different files - our agent log file and syslog file

Regards,
Deepak

On Mon, May 2, 2016 at 4:16 PM, njagadish <notifications@github.com> wrote:

> In gbpservice/nfp/service_vendor_agents/vyos/vyos_session/utils.py
> <https://github.com/oneconvergence/group-based-policy/pull/272#discussion_r61725110>
> :
>
> > @@ -68,6 +69,13 @@ def init_logger(logger):
> >      # add the handlers to the logger
> >      logger.addHandler(file_handler)
> >
> > +    formatter = logging.Formatter('vyos %(name)s %(funcName)s() %(levelname)s '
> > +                                  '%(message)s')
> > +    sys_handler = handlers.SysLogHandler(address=('localhost', 514))
> > +    sys_handler.setFormatter(formatter)
> > +    sys_handler.setLevel(logging.DEBUG)
> > +    logger.addHandler(sys_handler)
>
> There is an existing file handler and we added this sys_handler. Are the
> logs now landing in two different files or single one?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/oneconvergence/group-based-policy/pull/272/files/5355935019dfd5d052b87f1642c2883137a3dffa#r61725110>
>